From 4566d8cb91486d9e50cb991cad16b4849c22166e Mon Sep 17 00:00:00 2001
From: Dmitry Nikonov <nikonovds@gmail.com>
Date: Wed, 24 Jul 2019 02:45:08 +0300
Subject: [PATCH] Add missing device id for MX Master

Update DeviceMatch for Logitech MX Master in according with the libratbag device description (there are two ids for usb and two for bluetooth).
---
 data/svgs/svg-lookup.ini | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/svgs/svg-lookup.ini b/data/svgs/svg-lookup.ini
index f657d88..f7bc036 100644
--- a/data/svgs/svg-lookup.ini
+++ b/data/svgs/svg-lookup.ini
@@ -111,7 +111,7 @@ DeviceMatch=bluetooth:046d:b019;usb:046d:4069
 Svg=logitech-mx-master-2s.svg
 
 [Logitech MX Master]
-DeviceMatch=usb:046d:4041;bluetooth:046d:b012;usb:046d:4060
+DeviceMatch=usb:046d:4041;bluetooth:046d:b012;usb:046d:4060;bluetooth:046d:b017
 Svg=logitech-mx-master.svg
 
 [Roccat Kone XTD]
